packages/react-button/src/components/MenuButton/MenuButton.types.ts

```diff
@@ -0,0 +1,55 @@
+import * as React from 'react';
+import { BaseSlots, SlotProp, SlotProps } from '@fluentui/react-compose';
+import { ButtonProps, ButtonTokens } from '../Button/Button.types';
+
+export interface MenuButtonProps extends Omit<ButtonProps, 'icon' | 'iconPosition' | 'loader'> {
+  /**
+   * Shorthand menu that is displayed when the button is pressed.
+   */
+  // tslint:disable-next-line:no-any
+  menu?: SlotProp<any>;
```

Comment:
shouldn't here be `SlotProp<IContextualMenuProps>`?

Reply:
I don't think we want to marry `Button` and `ContextualMenu` just yet. Specially since there is a new `Menu` planned somewhere in the future. Happy to discuss this as well.

Reply:
but right now it is using `ContextualMenu`. so we should just use the correct typing? we can also change it when we actually using a new menu

Reply:
I'll make the change for now since we can revisit later as you said.